roles/web-server.json

```diff
@@ -4,25 +4,30 @@
   },
   "json_class": "Chef::Role",
   "run_list": [
+    "recipe[vim]",
     "recipe[git]",
-    "recipe[apt]",
+    "recipe[yum]",
+    "recipe[yum-epel]",
+    "recipe[selinux::permissive]",
```

Comment:
This ought to be unnecessary, but I wouldn't want to deal with it in a one-day training session, either.